libs/client/src/lib.rs

```diff
@@ -621,7 +621,7 @@ impl KanidmClient {
             .map_err(|e| ClientError::JsonDecode(e, opid))
     }
 
-    async fn perform_post_request<R: Serialize, T: DeserializeOwned>(
+    pub async fn perform_post_request<R: Serialize, T: DeserializeOwned>(
```

Comment:
Why make this public? Is there a client that wants to consume the wrapper directly?

Reply:
Yeah, I'm using it to test sending bad things (with embedded auth) over here: https://github.com/yaleman/kanidm/blob/6a8361d8d520e73753af2fe944af7393a88b42d5/server/testkit/tests/proto_v1_test.rs#L644 (and similar tests).